README.md

```diff
@@ -39,4 +39,6 @@ You can now view the development server at `localhost:8080`.
 - Twitter: [@swengineermwi](https://twitter.com/swengineermwi)
 - Linkedin: [Mwila Kaunda](https://www.linkedin.com/in/mwila-kaunda-5479b2168)
 
-## Show your support
\ No newline at end of file
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Kindly use better and more descriptive commit messages in future commits. Single words like "components" and "tests" are not descriptive.